runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -77,6 +87,7 @@ int32_t chpl_comm_getMaxThreads(void) {
 }
 
 void chpl_comm_init(int *argc_p, char ***argv_p) {
+  atomic_init_bool(&progress_threads_please_exit, false);
```

Comment:
Need to call `atomic_destroy_*` on all these atomics at some point (only really matters for the locks implementation, but still) -- related to https://github.com/chapel-lang/chapel/issues/5482

Reply:
Done, and thanks!